### DISCUSSION TITLE
any ideas what badges should be integrated / achievements ? => let me know below

### DISCUSSION THREAD

**Discussion by Unlucky-Life** [Unlucky-Life/ankimon#8] [Ideas]
Right now there are only like 10 Badges/ possible Achievements => any suggestions ?

**Comment by omfgroflmfaol** [comment]
<details> 
  <summary>(spoiler!) Click the triangle on the left to see a list of all the badges that are currently available (spoiler!)</summary>
   <img width="292" alt="image" src="https://github.com/Unlucky-Life/ankimon/assets/166659632/1e4ea2be-48f1-4d20-9626-f9928060876f">
</details>

Off the top of my head, I would suggest adding badges for
- caught a first of a type (fire, psychic etc.)
- catch em all: encountered / caught every pokemon of a gen (= completing the Pokédex of each gen)
- catch em all (for real this time): encountered / caught all pokemon (= completing the combine megadex)
- overpowered: defeated a pokemon in the first round of battle
- 3,5,10,... KO-streaks in one session (defeated x pokemon in one session without switching main pokemon)
- evolve a pokemon with an item
- status effect related achievements: keep an opposing pokemon burning for 10 moves in a row ("on fire"), putting a pokemon to sleep ("lullaby"), ...
- collector I, II, III, IV, V: Catch 25, 50, 100, 250, 500 different pokemon
- if at some point money is added to ankimon: rich kid (have a lot of money)
- have a pokemon reach lvl 100
- train your (a) starter pokemon to level 100

Also maybe have some achievements award you an item, e.g. the achievement for catching a poison type pokemon could give you an antidote.

Finding suitable sprites for the achievements might be an issue, although the available sprites from all the generations have some icons that might be repurposed. Gym badges could be utilized for the corresponding pokemon type, although I am not sure whether every type has a corresponding gym in pokemon history.
